.github/workflows/release.yml

```diff
@@ -498,6 +478,40 @@ jobs:
           asset_name: enso-launcher-${{ env.DIST_VERSION }}-windows-amd64.zip
           asset_content_type: application/zip
 
+      - name: Publish the Project Manager (Linux)
```

Comment:
Shouldn't these check what platform is being run?

Reply:
No, this is a separate job that is composing the packages and it is only running on a Linux runner - it uses artifacts built in an earlier stage on different platforms and packages them.